src/openrpc/media_access.json

```diff
@@ -0,0 +1,580 @@
+        "description": "A module for access user-cultivated media files, e.g. photos on a USB drive.",
+        "version": "0.0.0"
+    },
+    "methods": [
+        {
+            "name": "volumes",
+            "tags": [
+                {
+                    "name": "temporal-set"
+                },
+                {
+                    "name": "capabilities",
+                    "x-uses": [
+                        "xrn:firebolt:capability:media-access:volumes"
+                    ]
+                }
+            ],
+            "summary": "Access available Volumes with MediaFiles.",
+            "params": [
+                {
+                    "name": "query",
+                    "required": true,
+                    "schema": {
+                        "$ref": "#/components/schemas/Volume"
+                    }
+                }
+            ],
+            "result": {
+                "name": "volumes",
+                "schema": {
+                    "type": "array",
+                    "items": {
+                        "title": "Volume",
+                        "$ref": "#/components/schemas/Volume"
+                    }
+                }
+            },
+            "examples": [
+                {
+                    "name": "Default example #1",
+                    "params": [
+                        {
+                            "name": "query",
+                            "value": {
+                                "type": "all"
+                            }
+                        }
+                    ],
+                    "result": {
+                        "name": "volumes",
+                        "value": [
+                            {
+                                "uri": "/usb/my-usb-drive",
+                                "name": "My USB Drive",
+                                "type": "usb"
+                            },
+                            {
+                                "uri": "/local/photos",
+                                "name": "photos",
+                                "type": "local"
+                            }
+                        ]
+                    }
+                }
+            ]
+        },
+        {
+            "name": "audio",
+            "tags": [
+                {
+                    "name": "capabilities",
+                    "x-uses": [
+                        "xrn:firebolt:capability:media-access:audio"
+                    ]
+                }
+            ],
+            "summary": "Access available audio MediaFiles.",
+            "params": [
+                {
+                    "name": "volumes",
+                    "summary": "A set of queries used to match which volumes will be matched.",
+                    "required": false,
+                    "schema": {
+                        "$ref": "#/components/schemas/Volume"
+                    }
+                },
+                {
+                    "name": "files",
```

Comment:
the params are called `files` and `volumes` but the parameter is only a single Volume or MediaFile, I found that confusing.

Reply:
it's plural because it's a filter, so you're filtering which files you want. maybe I should rename the params to make that more clear.